Portfolio Mobile version

AztkMarty opened the pull request:
## In this Milestone, I introduced the following changes
-In this pull request, I have made some changes to the code to fix some bugs that were identified during local testing.
-The changes made include replacing the image used for the navigation menu with the original icon of the page and fixing the spacing above to improve the visual consistency of the design.
-Additionally, I corrected the gaps between the social media icons by adjusting the spacing with CSS, making them appear more evenly spaced and centered on the Y axis.

AztkMarty committed:
Added Linters

joel-kalema commented on style.css:
```diff
@@ -66,11 +65,11 @@
 /* BODY SECTION */
 
 /* Body styling */
+
 body {
   width: 381px;
   height: 6251px;
   margin: auto;
```
- [x] Good job working on the mobile version 👍, however, make sure you use `font-family: 'Inter';` to stick to the design as it is required

joel-kalema commented on style.css:
```diff
@@ -66,11 +65,11 @@
 /* BODY SECTION */
 
 /* Body styling */
+
 body {
   width: 381px;
```
- [x] Also, make sure you fix the `background image` to stick to the design as it is required 😃 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-04-30 020656](https://user-images.githubusercontent.com/57408419/235329183-a7f0df13-1671-4a3b-98da-49f4400418d6.png)| |![Screenshot 2023-04-30 020729](https://user-images.githubusercontent.com/57408419/235329187-ffd4fbb5-d65f-47e6-b5bf-0dcef8f61872.png)| |

- [x] Kindly, make sure you fix the color for technologies, to stick to the design as it is required. 😃, it suppose to be `
#36B37E`. Also `margin` and `font-size` 👍  

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-04-30 020921](https://user-images.githubusercontent.com/57408419/235329217-12c72c6b-52b8-4930-a827-617cb58f8d64.png)| |![Screenshot 2023-04-30 020819](https://user-images.githubusercontent.com/57408419/235329231-844ec39b-436b-4288-9476-dd49a5ce30c4.png)| |

joel-kalema reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @AztkMarty 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Added all changes from feedback

AztkMarty committed:
Added all changes from feedback

AztkMarty committed:
Added all changes from feedback and errors fixed

AztkMarty commented:
-Added the required font for the template.
-Changed gaps and rows spliced.
-Added vector figures and corrected their position according to the template.
-Added the correct color to the card and verified that everything was in line with the template's color scheme.
-Checked for errors locally.

didierganthier reviewed (approved):
Hi @AztkMarty ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

<img src="https://media.giphy.com/media/l9CSJHK88kMTe/giphy.gif" />

## Highlights
- Fixed background ✔️ 
- Added gaps ✔️ 
- Redesigned badges ✔️ 

## Optional Suggestions 💡
- N/A

Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉
